applications/models.py

```diff
@@ -130,6 +133,18 @@ class BootcampApplication(TimestampedModel):
     )
     state = FSMField(default=AppStates.AWAITING_PROFILE_COMPLETION.value, choices=VALID_APP_STATE_CHOICES)
 
+    @transition(
+        field=state,
+        source=[AppStates.AWAITING_RESUME, AppStates.AWAITING_USER_SUBMISSIONS],
+        target=AppStates.AWAITING_USER_SUBMISSIONS
+    )
+    def upload_resume(self, resume_file):
+        """Save resume and make sure that the state can be transitioned to a new state"""
+        validate_file_extension(resume_file)
+        self.resume_file = resume_file
+        self.resume_upload_date = datetime.datetime.now()
```

Comment:
Can we use `main.utils.now_in_utc` here so we have the time zone infomation included?